--- originals/adaryn.vim	2010-04-11 09:37:40.000000000 -0700
+++ patched/adaryn.vim	2010-04-15 10:50:13.000000000 -0700
@@ -70,3 +70,22 @@
 hi Ignore	guifg=bg ctermfg=bg
 
 
+hi CursorColumn guibg=grey40
+hi CursorLine gui=NONE guibg=grey40
+hi Error guibg=red guifg=white
+hi FoldColumn gui=NONE
+hi Folded gui=NONE
+hi MatchParen guibg=darkcyan
+hi MoreMsg gui=bold guifg=seagreen
+hi NonText gui=bold
+hi Pmenu guibg=magenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=darkgrey
+hi Search gui=NONE
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad gui=undercurl guisp=red
+hi SpellCap gui=undercurl guisp=blue
+hi SpellLocal gui=undercurl guisp=cyan
+hi SpellRare gui=undercurl guisp=magenta
+hi TabLine guibg=darkgrey guifg=NONE
+hi Underlined guifg=#80a0ff
