--- originals/adrian.vim	2010-04-05 14:44:38.000000000 -0700
+++ patched/adrian.vim	2010-04-10 21:46:36.000000000 -0700
@@ -95,3 +95,34 @@
 "highlight link SpecialComment  Special
 highlight link Debug           Special
 
+hi FoldColumn guibg=grey guifg=cyan
+hi SpellBad gui=undercurl guisp=red
+hi Folded guibg=darkgrey guifg=cyan
+hi Pmenu guibg=magenta
+hi MoreMsg gui=bold guifg=seagreen
+hi SpellCap gui=undercurl guisp=blue
+hi PmenuSel guibg=darkgrey
+hi NonText gui=bold guifg=blue
+hi LineNr guifg=yellow
+hi Title gui=bold guifg=magenta
+hi SpellLocal gui=undercurl guisp=cyan
+hi Underlined guifg=#80a0ff
+hi DiffDelete guibg=darkcyan gui=bold guifg=blue
+hi SpellRare gui=undercurl guisp=magenta
+hi TabLine guibg=darkgrey guifg=NONE
+hi PmenuSbar guibg=grey
+hi Question gui=bold guifg=green
+hi WildMenu guibg=yellow guifg=black
+hi SignColumn guibg=grey guifg=cyan
+hi lCursor guibg=fg guifg=bg
+hi Directory guifg=cyan
+hi CursorLine guibg=grey40 gui=NONE
+hi Identifier gui=NONE
+hi DiffText guibg=red
+hi SpecialKey guifg=cyan
+hi DiffChange guibg=darkmagenta
+hi WarningMsg guifg=red
+hi MatchParen guibg=darkcyan
+hi DiffAdd guibg=darkblue
+hi ErrorMsg guibg=red guifg=white
+hi CursorColumn guibg=grey40
