--- originals/aiseered.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/aiseered.vim	2010-04-15 11:12:50.000000000 -0700
@@ -35,3 +35,38 @@
 hi SpecialChar guifg=#ffaa00
 hi StorageClass guifg=#ddaacc
 hi Error guifg=red guibg=white
+
+hi CursorColumn guibg=grey40
+hi CursorLine gui=NONE guibg=grey40
+hi DiffAdd guibg=darkblue
+hi DiffChange guibg=darkmagenta
+hi DiffDelete gui=bold guibg=darkcyan guifg=blue
+hi DiffText guibg=red
+hi Directory guifg=cyan
+hi ErrorMsg guifg=white
+hi FoldColumn guibg=grey guifg=cyan
+hi Folded guibg=darkgrey guifg=cyan
+hi Identifier gui=NONE
+hi Ignore guifg=bg
+hi LineNr guifg=yellow
+hi MatchParen guibg=darkcyan
+hi MoreMsg gui=bold guifg=seagreen
+hi NonText gui=bold guifg=blue
+hi Pmenu guibg=magenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=darkgrey
+hi Question gui=bold guifg=green
+hi SignColumn guibg=grey guifg=cyan
+hi SpecialKey guifg=cyan
+hi SpellBad gui=undercurl guisp=red
+hi SpellCap gui=undercurl guisp=blue
+hi SpellLocal gui=undercurl guisp=cyan
+hi SpellRare gui=undercurl guisp=magenta
+hi TabLine guibg=darkgrey guifg=NONE
+hi Title gui=bold guifg=magenta
+hi Todo guibg=yellow guifg=blue
+hi Underlined guifg=#80a0ff
+hi Visual guibg=darkgrey
+hi WarningMsg guifg=red
+hi WildMenu guibg=yellow guifg=black
+hi lCursor guibg=fg guifg=bg
