--- originals/BlackSea.vim	2010-04-12 17:13:29.000000000 -0700
+++ patched/BlackSea.vim	2010-04-21 12:30:19.000000000 -0700
@@ -7,6 +7,8 @@
 "
 " Only values that differ from defaults are specified.
 
+" VIM-XTERM-TAG
+
 set background=dark
 hi clear
 if exists("syntax_on")
@@ -35,3 +37,47 @@
 " Original values:
 "hi Constant guifg=DeepPink
 "hi PreProc guifg=Magenta ctermfg=Magenta
+
+hi Constant guifg=#ffa0a0
+hi Cursor guibg=fg guifg=bg
+hi CursorColumn guibg=grey40
+hi CursorLine gui=NONE guibg=grey40
+hi DiffAdd guifg=NONE
+hi DiffChange guifg=NONE
+hi DiffDelete gui=bold
+hi DiffText guifg=NONE
+hi Directory guifg=cyan
+hi Error guibg=red guifg=white
+hi ErrorMsg guibg=red guifg=white
+hi FoldColumn guifg=cyan
+hi Folded guifg=cyan
+hi Identifier gui=NONE
+hi Ignore guifg=bg
+hi LineNr guifg=yellow
+hi MatchParen guibg=darkcyan
+hi MoreMsg gui=bold guifg=seagreen
+hi NonText gui=bold
+hi Pmenu guibg=magenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=darkgrey
+hi PreProc guifg=#ff80ff
+hi Question gui=bold guifg=green
+hi Search guifg=black
+hi SignColumn guibg=grey guifg=cyan
+hi Special guifg=orange
+hi SpecialKey guifg=cyan
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi Statement gui=bold
+hi StatusLine gui=bold,reverse
+hi TabLine guibg=darkgrey guifg=NONE
+hi Title gui=bold guifg=magenta
+hi Todo guibg=yellow guifg=blue
+hi Type guifg=#60ff60
+hi Underlined guifg=#80a0ff
+hi Visual guibg=darkgrey
+hi WarningMsg guifg=red
+hi WildMenu guibg=yellow guifg=black
+hi lCursor guibg=fg guifg=bg
