--- originals/bluegreen.vim	2009-06-04 21:52:52.000000000 -0700
+++ patched/bluegreen.vim	2010-04-19 22:52:02.000000000 -0700
@@ -14,7 +14,7 @@
 if exists("syntax_on")
     syntax reset
 endif
-let g:colors_name="mine"
+let g:colors_name="bluegreen"
 
 hi Normal       guifg=White guibg=#061A3E
 
@@ -47,4 +47,35 @@
 hi Ignore       guifg=grey60
 hi Todo guibg=#9C8C84 guifg=#244C0A
 
+hi CursorColumn guibg=grey40
+hi CursorLine gui=NONE guibg=grey40
+hi DiffAdd guibg=darkblue
+hi DiffChange guibg=darkmagenta
+hi DiffDelete gui=bold guibg=darkcyan guifg=blue
+hi DiffText guibg=red
+hi Directory guifg=cyan
+hi Error guibg=red guifg=white
+hi ErrorMsg guibg=red guifg=white
+hi Identifier gui=NONE
+hi LineNr guifg=yellow
+hi MatchParen guibg=darkcyan
+hi MoreMsg gui=bold
+hi NonText gui=bold
+hi Pmenu guibg=magenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=darkgrey
+hi Question gui=bold
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi Statement gui=bold
+hi TabLine guibg=darkgrey guifg=NONE
+hi Title gui=bold
+hi Type gui=bold
+hi Underlined guifg=#80a0ff
+hi WildMenu guibg=yellow guifg=black
+hi lCursor guibg=fg guifg=bg
+
 "vim: ts=4
