--- originals/breeze.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/breeze.vim	2010-04-20 09:06:08.000000000 -0700
@@ -68,3 +68,16 @@
 hi Todo         gui=BOLD,UNDERLINE guifg=#ffb0b0 guibg=NONE
 hi Type         gui=NONE guifg=#80ffa0 guibg=NONE
 hi Underlined   gui=UNDERLINE guifg=#ffffff guibg=NONE
+
+hi CursorColumn guibg=grey40
+hi CursorLine gui=NONE guibg=grey40
+hi MatchParen guibg=darkcyan
+hi Pmenu guibg=magenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=darkgrey
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi TabLine guibg=darkgrey guifg=NONE
