--- originals/brookstream.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/brookstream.vim	2010-04-15 11:20:51.000000000 -0700
@@ -80,4 +80,30 @@
 "hi link Debug   Special
 "hi link FoldColumn  Folded
 
+hi CursorColumn guibg=grey40
+hi CursorLine gui=NONE guibg=grey40
+hi DiffDelete gui=bold
+hi FoldColumn guibg=grey guifg=cyan
+hi Folded guibg=darkgrey
+hi Identifier gui=NONE
+hi MatchParen guibg=darkcyan
+hi MoreMsg gui=bold
+hi NonText gui=bold
+hi Pmenu guibg=magenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=darkgrey
+hi Question gui=bold
+hi Search guibg=yellow guifg=black
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad gui=undercurl guisp=red
+hi SpellCap gui=undercurl guisp=blue
+hi SpellLocal gui=undercurl guisp=cyan
+hi SpellRare gui=undercurl guisp=magenta
+hi Statement gui=bold
+hi TabLine guibg=darkgrey guifg=NONE
+hi Title gui=bold
+hi Type gui=bold
+hi WildMenu guibg=yellow guifg=black
+hi lCursor guibg=fg guifg=bg
+
 "- end of colorscheme -----------------------------------------------	
