--- originals/buttercream.vim	2009-06-04 20:53:28.000000000 -0700
+++ patched/buttercream.vim	2010-04-21 12:32:41.000000000 -0700
@@ -7,6 +7,8 @@
 " As of now only gui is supported
 " Based on the theme fog theme by Thomas R. Kimpton <tomk@emcity.net>
 
+" VIM-XTERM-TAG
+
 set background=light
 hi clear
 if exists("syntax_on")
@@ -57,3 +59,21 @@
 hi Visual               guifg=LightGrey                                         gui=reverse
 hi VisualNOS                                                                    gui=underline,bold
 hi Todo                                         guibg=#ccaa22                   gui=bold,underline
+
+hi CursorColumn guibg=grey90
+hi CursorLine gui=NONE guibg=grey90
+hi Error guibg=red guifg=white
+hi MatchParen guibg=cyan
+hi Pmenu guibg=lightmagenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=grey
+hi SignColumn guibg=grey guifg=darkblue
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi TabLine guibg=lightgrey guifg=NONE
+hi Todo guifg=blue
+hi Type gui=bold
+hi Underlined guifg=slateblue
+hi Visual guibg=lightgrey
