--- originals/camo.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/camo.vim	2010-04-20 09:13:37.000000000 -0700
@@ -74,3 +74,24 @@
 hi Error        guifg=bisque    guibg=red
 hi Todo         guifg=red       guibg=bisque
 
+hi CursorColumn guibg=grey40
+hi CursorLine gui=NONE guibg=grey40
+hi DiffDelete gui=bold
+hi Identifier gui=NONE
+hi MatchParen guibg=darkcyan
+hi MoreMsg gui=bold
+hi NonText gui=bold
+hi Pmenu guibg=magenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=darkgrey
+hi Question gui=bold
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi Statement gui=bold
+hi TabLine guibg=darkgrey guifg=NONE
+hi Title gui=bold
+hi Type gui=bold
+hi lCursor guibg=fg guifg=bg
