--- originals/camo.vim	2010-01-20 23:43:36.000000000 -0800
+++ patched/camo.vim	2010-04-21 12:34:12.000000000 -0700
@@ -2,6 +2,8 @@
 " Maintainer:   Tim Aldrich <aldy0169@yahoo.com>
 " Last Change:  19 January 2002
 
+" VIM-XTERM-TAG
+
 set background=dark
 hi clear
 if exists("syntax_on")
@@ -74,3 +76,24 @@
 hi Error        guifg=bisque    guibg=red
 hi Todo         guifg=red       guibg=bisque
 
+hi CursorColumn guibg=grey40
+hi CursorLine gui=NONE guibg=grey40
+hi DiffDelete gui=bold
+hi Identifier gui=NONE
+hi MatchParen guibg=darkcyan
+hi MoreMsg gui=bold
+hi NonText gui=bold
+hi Pmenu guibg=magenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=darkgrey
+hi Question gui=bold
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi Statement gui=bold
+hi TabLine guibg=darkgrey guifg=NONE
+hi Title gui=bold
+hi Type gui=bold
+hi lCursor guibg=fg guifg=bg
