--- originals/candy.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/candy.vim	2010-04-20 09:16:10.000000000 -0700
@@ -76,3 +76,16 @@
 hi htmlItalic               gui=ITALIC
 hi htmlUnderline            gui=UNDERLINE
 hi htmlUnderlineItalic      gui=UNDERLINE,ITALIC
+
+hi CursorColumn guibg=grey40
+hi CursorLine gui=NONE guibg=grey40
+hi MatchParen guibg=darkcyan
+hi Pmenu guibg=magenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=darkgrey
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi TabLine guibg=darkgrey guifg=NONE
