--- originals/cleanphp.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/cleanphp.vim	2010-04-11 09:52:13.000000000 -0700
@@ -78,4 +78,22 @@
 hi link SpecialComment	Special
 hi link Debug			Special
 
-" vim: sw=2
\ No newline at end of file
+hi CursorColumn guibg=grey40
+hi CursorLine gui=NONE guibg=grey40
+hi Error guibg=red guifg=white
+hi Identifier gui=NONE
+hi MatchParen guibg=darkcyan
+hi Pmenu guibg=magenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=darkgrey
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad gui=undercurl guisp=red
+hi SpellCap gui=undercurl guisp=blue
+hi SpellLocal gui=undercurl guisp=cyan
+hi SpellRare gui=undercurl guisp=magenta
+hi TabLine guibg=darkgrey guifg=NONE
+hi Todo guibg=yellow guifg=blue
+hi Type gui=bold
+hi Underlined guifg=#80a0ff
+
+" vim: sw=2
