--- originals/dante.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/dante.vim	2010-04-20 09:36:50.000000000 -0700
@@ -6,6 +6,7 @@
 """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 " Colorscheme Option:
 """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+set background=dark
 hi clear
 if exists("syntax on")
 	syntax reset
@@ -80,4 +81,19 @@
 hi link Delimiter	Special
 hi link SpecialComment	Special
 hi link Debug		Special
+
+hi CursorColumn guibg=grey40
+hi CursorLine gui=NONE guibg=grey40
+hi Identifier gui=NONE
+hi MatchParen guibg=darkcyan
+hi NonText gui=bold
+hi Pmenu guibg=magenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=darkgrey
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi TabLine guibg=darkgrey guifg=NONE
 """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
