--- originals/darkspectrum.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/darkspectrum.vim	2010-04-20 09:44:05.000000000 -0700
@@ -51,7 +51,7 @@
 hi LineNr		guifg=#535353 guibg=#202020
 hi NonText		guifg=#535353 guibg=#202020
 hi Folded		guifg=#535353 guibg=#202020 gui=bold
-hi FoldeColumn  guifg=#535353 guibg=#202020 gui=bold
+hi FoldColumn  guifg=#535353 guibg=#202020 gui=bold
 "hi VertSplit	guibg=#ffffff guifg=#ffffff gui=none
 
 hi VertSplit	guibg=#3C3C3C guifg=#3C3C3C gui=none
@@ -102,16 +102,16 @@
 hi spellLocal   guisp=#729fcf
 
 hi link cppSTL          Function
-hi link Error			Todo
+hi! link Error			Todo
 hi link Character		Number
 hi link rubySymbol		Number
 hi link htmlTag			htmlEndTag
 "hi link htmlTagName     htmlTag
 hi link htmlLink		Underlined
 hi link pythonFunction	Identifier
-hi link Question		Type
+hi! link Question		Type
 hi link CursorIM		Cursor
-hi link VisualNOS		Visual
+"hi! link VisualNOS		Visual
 hi link xmlTag			Identifier
 hi link xmlTagName		Identifier
 hi link shDeref			Identifier
@@ -127,4 +127,17 @@
 hi TabLineFill	guifg=#535353 guibg=#202020 gui=none
 hi TabLineSel	guifg=#FFFFFF gui=bold
 "hi TabLineSel	guifg=#FFFFFF guibg=#000000 gui=bold
+
+hi CursorLine gui=NONE
+hi ErrorMsg guibg=red guifg=white
+hi Identifier gui=NONE
+hi MoreMsg gui=bold
+hi NonText gui=bold
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi Title gui=bold
+hi lCursor guibg=fg guifg=bg
 " vim: sw=4 ts=4
