--- originals/darkbone.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/darkbone.vim	2010-04-20 09:40:48.000000000 -0700
@@ -99,4 +99,15 @@
   hi TabLineSel	  guifg=#c0c0ff guibg=#606080 gui=bold
 endif
 
+hi CursorLine gui=NONE
+hi Identifier gui=NONE
+hi Ignore guifg=bg
+hi MoreMsg gui=bold
+hi Question gui=bold
+hi SignColumn guifg=cyan
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+
 " vim:set ts=8 sts=2 sw=2 tw=0:
