--- originals/darkspectrum.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/darkspectrum.vim	2010-04-11 11:05:21.000000000 -0700
@@ -51,7 +51,7 @@
 hi LineNr		guifg=#535353 guibg=#202020
 hi NonText		guifg=#535353 guibg=#202020
 hi Folded		guifg=#535353 guibg=#202020 gui=bold
-hi FoldeColumn  guifg=#535353 guibg=#202020 gui=bold
+hi FoldColumn  guifg=#535353 guibg=#202020 gui=bold
 "hi VertSplit	guibg=#ffffff guifg=#ffffff gui=none
 
 hi VertSplit	guibg=#3C3C3C guifg=#3C3C3C gui=none
@@ -101,30 +101,43 @@
 hi spellRare    guisp=#ad7fa8
 hi spellLocal   guisp=#729fcf
 
-hi link cppSTL          Function
-hi link Error			Todo
-hi link Character		Number
-hi link rubySymbol		Number
-hi link htmlTag			htmlEndTag
-"hi link htmlTagName     htmlTag
-hi link htmlLink		Underlined
-hi link pythonFunction	Identifier
-hi link Question		Type
-hi link CursorIM		Cursor
-hi link VisualNOS		Visual
-hi link xmlTag			Identifier
-hi link xmlTagName		Identifier
-hi link shDeref			Identifier
-hi link shVariable		Function
-hi link rubySharpBang	Special
-hi link perlSharpBang	Special
-hi link schemeFunc      Statement
-"hi link shSpecialVariables Constant
-"hi link bashSpecialVariables Constant
+hi! link cppSTL          Function
+hi! link Error			Todo
+hi! link Character		Number
+hi! link rubySymbol		Number
+hi! link htmlTag			htmlEndTag
+"hi! link htmlTagName     htmlTag
+hi! link htmlLink		Underlined
+hi! link pythonFunction	Identifier
+hi! link Question		Type
+hi! link CursorIM		Cursor
+hi! link VisualNOS		Visual
+hi! link xmlTag			Identifier
+hi! link xmlTagName		Identifier
+hi! link shDeref			Identifier
+hi! link shVariable		Function
+hi! link rubySharpBang	Special
+hi! link perlSharpBang	Special
+hi! link schemeFunc      Statement
+"hi! link shSpecialVariables Constant
+"hi! link bashSpecialVariables Constant
 
 " tabs (non gui)
 hi TabLine		guifg=#A3A3A3 guibg=#202020 gui=none
 hi TabLineFill	guifg=#535353 guibg=#202020 gui=none
 hi TabLineSel	guifg=#FFFFFF gui=bold
 "hi TabLineSel	guifg=#FFFFFF guibg=#000000 gui=bold
+
+hi CursorLine gui=NONE
+hi ErrorMsg guibg=red guifg=white
+hi Identifier gui=NONE
+hi MoreMsg gui=bold
+hi NonText gui=bold
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad gui=undercurl guisp=#fcaf3e
+hi SpellCap gui=undercurl guisp=#73d216
+hi SpellLocal gui=undercurl guisp=#729fcf
+hi SpellRare gui=undercurl guisp=#ad7fa8
+hi Title gui=bold
+hi lCursor guibg=fg guifg=bg
 " vim: sw=4 ts=4
