--- originals/eclipse.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/eclipse.vim	2010-04-20 10:17:25.000000000 -0700
@@ -46,10 +46,10 @@
 highlight FoldColumn gui=none guifg=#6b6b6b guibg=#e7e7e7 ctermfg=black ctermbg=white
 
 " Popup Menu
-highlight PMenu      ctermbg=green ctermfg=white
-highlight PMenuSel   ctermbg=white ctermfg=black
-highlight PMenuSBar  ctermbg=red   ctermfg=white
-highlight PMenuThumb ctermbg=white ctermfg=red
+highlight PMenu      guibg=green guifg=white
+highlight PMenuSel   guibg=white guifg=black
+highlight PMenuSBar  guibg=red   guifg=white
+highlight PMenuThumb guibg=white guifg=red
 
 " Other
 highlight Directory  gui=none guifg=#7050ff guibg=bg
@@ -74,19 +74,14 @@
 highlight String     gui=none guifg=#8010a0 guibg=bg      ctermfg=Yellow
 highlight Number     gui=none guifg=#0000ff guibg=bg      ctermfg=White
 
-if !has("gui_running")
-    hi link Float          Number
-    hi link Conditional    Repeat
-    hi link Include        PreProc
-    hi link Macro          PreProc
-    hi link PreCondit      PreProc
-    hi link StorageClass   Type
-    hi link Structure      Type
-    hi link Typedef        Type
-    hi link Tag            Special
-    hi link Delimiter      Normal
-    hi link SpecialComment Special
-    hi link Debug          Special
-endif
+hi CursorColumn guibg=grey40
+hi CursorLine gui=NONE guibg=grey40
+hi MatchParen guibg=darkcyan
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi TabLine guibg=darkgrey guifg=NONE
 
 " vim:ff=unix:
