--- originals/eclipse.vim	2010-01-20 23:43:36.000000000 -0800
+++ patched/eclipse.vim	2010-04-13 00:20:57.000000000 -0700
@@ -45,12 +45,6 @@
 highlight Folded     gui=none guifg=#804030 guibg=#fff0d0 ctermbg=black ctermfg=black cterm=bold
 highlight FoldColumn gui=none guifg=#6b6b6b guibg=#e7e7e7 ctermfg=black ctermbg=white
 
-" Popup Menu
-highlight PMenu      ctermbg=green ctermfg=white
-highlight PMenuSel   ctermbg=white ctermfg=black
-highlight PMenuSBar  ctermbg=red   ctermfg=white
-highlight PMenuThumb ctermbg=white ctermfg=red
-
 " Other
 highlight Directory  gui=none guifg=#7050ff guibg=bg
 highlight LineNr     gui=none guifg=#6b6b6b guibg=#eeeeee
@@ -74,7 +68,6 @@
 highlight String     gui=none guifg=#8010a0 guibg=bg      ctermfg=Yellow
 highlight Number     gui=none guifg=#0000ff guibg=bg      ctermfg=White
 
-if !has("gui_running")
     hi link Float          Number
     hi link Conditional    Repeat
     hi link Include        PreProc
@@ -87,6 +80,18 @@
     hi link Delimiter      Normal
     hi link SpecialComment Special
     hi link Debug          Special
-endif
+
+hi CursorColumn guibg=grey90
+hi CursorLine gui=NONE guibg=grey90
+hi MatchParen guibg=cyan
+hi Pmenu guibg=lightmagenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=grey
+hi SignColumn guibg=grey guifg=darkblue
+hi SpellBad gui=undercurl guisp=red
+hi SpellCap gui=undercurl guisp=blue
+hi SpellLocal gui=undercurl guisp=darkcyan
+hi SpellRare gui=undercurl guisp=magenta
+hi TabLine guibg=lightgrey guifg=NONE
 
 " vim:ff=unix:
