--- originals/fine_blue.vim	2010-01-20 23:43:36.000000000 -0800
+++ patched/fine_blue.vim	2010-04-21 13:16:00.000000000 -0700
@@ -4,6 +4,8 @@
 "     version: 1.7
 " This color scheme uses a light background.
 
+" VIM-XTERM-TAG
+
 set background=light
 hi clear
 if exists("syntax_on")
@@ -69,3 +71,16 @@
 hi Todo         gui=NONE guifg=#ff0070 guibg=#ffe0f4
 hi Type         gui=NONE guifg=#7040ff guibg=NONE
 hi Underlined   gui=UNDERLINE guifg=#0000ff guibg=NONE
+
+hi CursorColumn guibg=grey90
+hi CursorLine gui=NONE guibg=grey90
+hi MatchParen guibg=cyan
+hi Pmenu guibg=lightmagenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=grey
+hi SignColumn guibg=grey guifg=darkblue
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi TabLine guibg=lightgrey guifg=NONE
