--- originals/fnaqevan.vim	2010-01-20 22:52:32.000000000 -0800
+++ patched/fnaqevan.vim	2010-04-21 13:16:19.000000000 -0700
@@ -4,6 +4,8 @@
 "
 " This color scheme uses a black (dark) background.
 
+" VIM-XTERM-TAG
+
 " First remove all existing highlighting.
 set background=dark
 hi clear
@@ -65,3 +67,17 @@
 hi rcursor      guibg=#00CCFF guifg=#000000 gui=NONE
 hi lcursor      guibg=#40D040 guifg=#000000 gui=NONE
 
+hi CursorColumn guibg=grey40
+hi CursorLine gui=NONE guibg=grey40
+hi Ignore guifg=bg
+hi MatchParen guibg=darkcyan
+hi Pmenu guibg=magenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=darkgrey
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi TabLine guibg=darkgrey guifg=NONE
+hi Underlined guifg=#80a0ff
