--- originals/habiLight.vim	2010-04-12 21:24:23.000000000 -0700
+++ patched/habiLight.vim	2010-04-18 20:56:01.000000000 -0700
@@ -118,7 +118,7 @@
     highlight SpellBad    term=reverse   ctermbg=12 gui=undercurl guisp=Red
     highlight SpellCap    term=reverse   ctermbg=9  gui=undercurl guisp=Blue
     highlight SpellRare   term=reverse   ctermbg=13 gui=undercurl guisp=Magenta
-    highlight SpellLocale term=underline ctermbg=11 gui=undercurl guisp=DarkCyan
+    highlight SpellLocal  term=underline ctermbg=11 gui=undercurl guisp=DarkCyan
 endif 
 
 " Completion {{{1
@@ -134,5 +134,15 @@
 
 highlight MatchParen  term=reverse ctermbg=11 gui=bold guibg=#B5EEB5 guifg=black
 
+hi CursorColumn guibg=grey90
+hi CursorLine gui=NONE guibg=grey90
+hi Ignore guifg=bg
+hi MoreMsg gui=bold
+hi SignColumn guibg=grey guifg=darkblue
+hi SpecialKey guifg=blue
+hi SpellLocal gui=undercurl guisp=darkcyan
+hi TabLine guifg=NONE
+hi Underlined guifg=slateblue
+hi lCursor guibg=fg guifg=bg
 
 " vim600:foldmethod=marker
