--- originals/habiLight.vim	2010-04-12 21:24:23.000000000 -0700
+++ patched/habiLight.vim	2010-04-21 15:23:12.000000000 -0700
@@ -10,6 +10,8 @@
 "              1.1: Vim 7 support added (completion, spell checker, paren, tabs)
 "              1.0: initial version
 "
+" VIM-XTERM-TAG
+
 " Intro {{{1
 set background=light
 hi clear
@@ -118,7 +120,7 @@
     highlight SpellBad    term=reverse   ctermbg=12 gui=undercurl guisp=Red
     highlight SpellCap    term=reverse   ctermbg=9  gui=undercurl guisp=Blue
     highlight SpellRare   term=reverse   ctermbg=13 gui=undercurl guisp=Magenta
-    highlight SpellLocale term=underline ctermbg=11 gui=undercurl guisp=DarkCyan
+    highlight SpellLocal  term=underline ctermbg=11 gui=undercurl guisp=DarkCyan
 endif 
 
 " Completion {{{1
@@ -134,5 +136,14 @@
 
 highlight MatchParen  term=reverse ctermbg=11 gui=bold guibg=#B5EEB5 guifg=black
 
+hi CursorColumn guibg=grey90
+hi CursorLine gui=NONE guibg=grey90
+hi Ignore guifg=bg
+hi MoreMsg gui=bold
+hi SignColumn guibg=grey guifg=darkblue
+hi SpecialKey guifg=blue
+hi TabLine guifg=NONE
+hi Underlined guifg=slateblue
+hi lCursor guibg=fg guifg=bg
 
 " vim600:foldmethod=marker
