--- originals/ir_black.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/ir_black.vim	2010-04-20 10:37:16.000000000 -0700
@@ -210,3 +210,22 @@
 
 
 " Special for PHP
+
+
+hi DiffAdd guibg=darkblue
+hi DiffChange guibg=darkmagenta
+hi DiffDelete gui=bold guibg=darkcyan guifg=blue
+hi DiffText guibg=red
+hi Directory guifg=cyan
+hi FoldColumn guibg=grey guifg=cyan
+hi Ignore guifg=bg
+hi MoreMsg gui=bold guifg=seagreen
+hi Question gui=bold guifg=green
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi TabLine guibg=darkgrey guifg=NONE
+hi Underlined guifg=#80a0ff
+hi lCursor guibg=fg guifg=bg
