--- originals/ir_black.vim	2010-04-03 10:26:49.000000000 -0700
+++ patched/ir_black.vim	2010-04-14 13:05:45.000000000 -0700
@@ -210,3 +210,22 @@
 
 
 " Special for PHP
+
+
+hi DiffAdd guibg=darkblue
+hi DiffChange guibg=darkmagenta
+hi DiffDelete gui=bold guibg=darkcyan guifg=blue
+hi DiffText guibg=red
+hi Directory guifg=cyan
+hi FoldColumn guibg=grey guifg=cyan
+hi Ignore guifg=bg
+hi MoreMsg gui=bold guifg=seagreen
+hi Question gui=bold guifg=green
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad gui=undercurl guisp=red
+hi SpellCap gui=undercurl guisp=blue
+hi SpellLocal gui=undercurl guisp=cyan
+hi SpellRare gui=undercurl guisp=magenta
+hi TabLine guibg=darkgrey guifg=NONE
+hi Underlined guifg=#80a0ff
+hi lCursor guibg=fg guifg=bg
