--- originals/ir_black.vim	2010-04-03 10:26:49.000000000 -0700
+++ patched/ir_black.vim	2010-04-11 16:26:03.000000000 -0700
@@ -123,23 +123,23 @@
 hi Delimiter        guifg=#00A0A0     guibg=NONE        gui=NONE      ctermfg=cyan        ctermbg=NONE        cterm=NONE
 hi Operator         guifg=white       guibg=NONE        gui=NONE      ctermfg=white       ctermbg=NONE        cterm=NONE
 
-hi link Character       Constant
-hi link Boolean         Constant
-hi link Float           Number
-hi link Repeat          Statement
-hi link Label           Statement
-hi link Exception       Statement
-hi link Include         PreProc
-hi link Define          PreProc
-hi link Macro           PreProc
-hi link PreCondit       PreProc
-hi link StorageClass    Type
-hi link Structure       Type
-hi link Typedef         Type
-hi link Tag             Special
-hi link SpecialChar     Special
-hi link SpecialComment  Special
-hi link Debug           Special
+hi! link Character       Constant
+hi! link Boolean         Constant
+hi! link Float           Number
+hi! link Repeat          Statement
+hi! link Label           Statement
+hi! link Exception       Statement
+hi! link Include         PreProc
+hi! link Define          PreProc
+hi! link Macro           PreProc
+hi! link PreCondit       PreProc
+hi! link StorageClass    Type
+hi! link Structure       Type
+hi! link Typedef         Type
+hi! link Tag             Special
+hi! link SpecialChar     Special
+hi! link SpecialComment  Special
+hi! link Debug           Special
 
 
 " Special for Ruby
@@ -158,27 +158,27 @@
 "rubyClassVariable
 "rubyBeginEnd
 "rubyRepeatModifier
-"hi link rubyArrayDelimiter    Special  " [ , , ]
+"hi! link rubyArrayDelimiter    Special  " [ , , ]
 "rubyCurlyBlock  { , , }
 
-hi link rubyClass             Keyword 
-hi link rubyModule            Keyword 
-hi link rubyKeyword           Keyword 
-hi link rubyOperator          Operator
-hi link rubyIdentifier        Identifier
-hi link rubyInstanceVariable  Identifier
-hi link rubyGlobalVariable    Identifier
-hi link rubyClassVariable     Identifier
-hi link rubyConstant          Type  
+hi! link rubyClass             Keyword 
+hi! link rubyModule            Keyword 
+hi! link rubyKeyword           Keyword 
+hi! link rubyOperator          Operator
+hi! link rubyIdentifier        Identifier
+hi! link rubyInstanceVariable  Identifier
+hi! link rubyGlobalVariable    Identifier
+hi! link rubyClassVariable     Identifier
+hi! link rubyConstant          Type  
 
 
 " Special for Java
-" hi link javaClassDecl    Type
-hi link javaScopeDecl         Identifier 
-hi link javaCommentTitle      javaDocSeeTag 
-hi link javaDocTags           javaDocSeeTag 
-hi link javaDocParam          javaDocSeeTag 
-hi link javaDocSeeTagParam    javaDocSeeTag 
+" hi! link javaClassDecl    Type
+hi! link javaScopeDecl         Identifier 
+hi! link javaCommentTitle      javaDocSeeTag 
+hi! link javaDocTags           javaDocSeeTag 
+hi! link javaDocParam          javaDocSeeTag 
+hi! link javaDocSeeTagParam    javaDocSeeTag 
 
 hi javaDocSeeTag              guifg=#CCCCCC     guibg=NONE        gui=NONE      ctermfg=darkgray    ctermbg=NONE        cterm=NONE
 hi javaDocSeeTag              guifg=#CCCCCC     guibg=NONE        gui=NONE      ctermfg=darkgray    ctermbg=NONE        cterm=NONE
@@ -186,27 +186,46 @@
 
 
 " Special for XML
-hi link xmlTag          Keyword 
-hi link xmlTagName      Conditional 
-hi link xmlEndTag       Identifier 
+hi! link xmlTag          Keyword 
+hi! link xmlTagName      Conditional 
+hi! link xmlEndTag       Identifier 
 
 
 " Special for HTML
-hi link htmlTag         Keyword 
-hi link htmlTagName     Conditional 
-hi link htmlEndTag      Identifier 
+hi! link htmlTag         Keyword 
+hi! link htmlTagName     Conditional 
+hi! link htmlEndTag      Identifier 
 
 
 " Special for Javascript
-hi link javaScriptNumber      Number 
+hi! link javaScriptNumber      Number 
 
 
 " Special for Python
-"hi  link pythonEscape         Keyword      
+"hi!  link pythonEscape         Keyword      
 
 
 " Special for CSharp
-hi  link csXmlTag             Keyword      
+hi!  link csXmlTag             Keyword      
 
 
 " Special for PHP
+
+
+hi DiffAdd guibg=darkblue
+hi DiffChange guibg=darkmagenta
+hi DiffDelete gui=bold guibg=darkcyan guifg=blue
+hi DiffText guibg=red
+hi Directory guifg=cyan
+hi FoldColumn guibg=grey guifg=cyan
+hi Ignore guifg=bg
+hi MoreMsg gui=bold guifg=seagreen
+hi Question gui=bold guifg=green
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad gui=undercurl guisp=red
+hi SpellCap gui=undercurl guisp=blue
+hi SpellLocal gui=undercurl guisp=cyan
+hi SpellRare gui=undercurl guisp=magenta
+hi TabLine guibg=darkgrey guifg=NONE
+hi Underlined guifg=#80a0ff
+hi lCursor guibg=fg guifg=bg
