--- originals/ironman.vim	2010-01-20 23:43:36.000000000 -0800
+++ patched/ironman.vim	2010-04-21 15:25:37.000000000 -0700
@@ -7,6 +7,8 @@
 " GUI only
 " inspired by colorsheme PYTE
 
+" VIM-XTERM-TAG
+
 set background=light
 hi clear
 if exists("syntax_on")
@@ -124,10 +126,15 @@
 highlight SpellBad           gui=undercurl guisp=Red
 highlight SpellCap           gui=undercurl guisp=Blue
 highlight SpellRare          gui=undercurl guisp=Magenta
-highlight SpellLocale        gui=undercurl guisp=DarkCyan
+highlight SpellLocal         gui=undercurl guisp=DarkCyan
 
 " Completion {{{1
 highlight Pmenu              guifg=Black   guibg=#BDDFFF
 highlight PmenuSel           guifg=Black   guibg=Orange
 highlight PmenuSbar          guifg=#CCCCCC guibg=#CCCCCC
 highlight PmenuThumb         gui=reverse guifg=Black   guibg=#AAAAAA
+
+hi Directory guifg=blue
+hi SignColumn guibg=grey guifg=darkblue
+hi TabLine guifg=NONE
+hi WildMenu guibg=yellow guifg=black
