--- originals/jellybeans.vim	2010-01-20 22:48:48.000000000 -0800
+++ patched/jellybeans.vim	2010-04-23 20:17:02.000000000 -0700
@@ -37,6 +37,8 @@
 " OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 " THE SOFTWARE.
 
+" VIM-XTERM-TAG
+
 set background=dark
 
 hi clear
@@ -269,20 +271,20 @@
 		if !l:fge && !l:bge
 			exec "hi ".a:group." guifg=#".a:fg." guibg=#".a:bg." ctermfg=".s:rgb(a:fg)." ctermbg=".s:rgb(a:bg)
 		elseif !l:fge && l:bge
-			exec "hi ".a:group." guifg=#".a:fg." guibg=NONE ctermfg=".s:rgb(a:fg)
+			exec "hi ".a:group." guifg=#".a:fg." guibg=NONE ctermfg=".s:rgb(a:fg)." ctermbg=NONE"
 		elseif l:fge && !l:bge
-			exec "hi ".a:group." guifg=NONE guibg=#".a:bg." ctermbg=".s:rgb(a:bg)
+			exec "hi ".a:group." guifg=NONE guibg=#".a:bg." ctermfg=NONE ctermbg=".s:rgb(a:bg)
 		endif
 	endif
 
 	if a:attr == ""
 		exec "hi ".a:group." gui=none cterm=none"
 	else
-		if a:attr == 'italic'
-			exec "hi ".a:group." gui=".a:attr." cterm=none"
-		else
-			exec "hi ".a:group." gui=".a:attr." cterm=".a:attr
+		let noitalic = join(filter(split(a:attr, ","), "v:val !=? 'italic'"), ",")
+		if empty(noitalic)
+			let noitalic = "none"
 		endif
+		exec "hi ".a:group." gui=".a:attr." cterm=".noitalic
 	endif
 endfun
 " }}}
@@ -343,7 +345,7 @@
 
 call s:X("Directory","dad085","","","","")
 call s:X("ErrorMsg","","902020","","","")
-hi link Error ErrorMsg
+hi! link Error ErrorMsg
 
 " Diff
 
@@ -397,6 +399,23 @@
 " Tag list
 hi link TagListFileName Directory
 
+" Originally missing
+call s:X("MoreMsg", "2e8b57", "", "bold", "", "")
+call s:X("WarningMsg", "ff0000", "", "", "", "")
+call s:X("Question", "00ff00", "", "bold", "", "")
+call s:X("Underlined", "80a0ff", "", "underline", "", "")
+hi SpellBad ctermbg=88
+hi SpellCap ctermbg=19
+hi SpellRare ctermbg=90
+hi SpellLocal ctermbg=30
+" These don't work well with the auto-conversion
+hi Folded ctermbg=238
+hi FoldColumn ctermbg=238
+hi SignColumn ctermbg=238
+hi DiffAdd ctermbg=22
+hi DiffDelete ctermbg=52
+hi DiffText ctermbg=17
+
 " delete functions {{{
 delf s:X
 delf s:rgb
