--- originals/leo.vim	2010-04-05 19:44:07.000000000 -0700
+++ patched/leo.vim	2010-04-11 17:00:13.000000000 -0700
@@ -58,8 +58,6 @@
         end
 endif
 
-let g:colors_name = "leo256"
-
 set background=dark
 if v:version > 580
         highlight clear
@@ -68,6 +66,8 @@
         endif
 endif
 
+let g:colors_name = "leo256"
+
 " Normal should come first
 hi Normal                               cterm=none              ctermfg=255             ctermbg=16 guibg=#000000 guifg=#ffffff
 hi CursorLine           cterm=none              ctermbg=16              guibg=#101010
@@ -148,3 +148,34 @@
 
 " ADDITIONNAL
 hi Repeat               cterm=none              ctermfg=142             guifg=#afaf00
+
+hi Character guifg=NONE
+hi Cursor guibg=fg guifg=bg
+hi CursorColumn guibg=grey40
+hi DiffDelete gui=bold
+hi Directory guifg=cyan
+hi Error guibg=red
+hi ErrorMsg guibg=red
+hi FoldColumn guifg=cyan
+hi Ignore guifg=bg
+hi MoreMsg gui=bold guifg=seagreen
+hi NonText gui=bold guifg=blue
+hi PmenuSbar guifg=NONE
+hi PmenuThumb gui=reverse guibg=NONE
+hi Question gui=bold guifg=green
+hi SignColumn guifg=cyan
+hi SpellBad guifg=NONE
+hi SpellCap gui=undercurl guifg=NONE guisp=blue
+hi SpellLocal gui=undercurl guifg=NONE guisp=cyan
+hi SpellRare gui=undercurl guifg=NONE guisp=magenta
+hi StatusLine gui=bold,reverse
+hi StatusLineNC gui=reverse
+hi TabLine gui=underline guibg=darkgrey guifg=NONE
+hi TabLineFill gui=reverse guibg=NONE guifg=NONE
+hi TabLineSel gui=bold guibg=NONE guifg=NONE
+hi Title gui=bold guifg=magenta
+hi VertSplit gui=reverse
+hi Visual gui=NONE guibg=darkgrey
+hi VisualNOS gui=bold,underline
+hi WildMenu guibg=yellow
+hi lCursor guibg=fg guifg=bg
