--- originals/matrix.vim	2009-06-04 20:53:28.000000000 -0700
+++ patched/matrix.vim	2010-04-21 15:32:25.000000000 -0700
@@ -5,6 +5,8 @@
 " Maintainer:   MURAOKA Taro <koron@tka.att.ne.jp>
 " Last Change:  10-Jun-2003.
 
+" VIM-XTERM-TAG
+
 set background=dark
 hi clear
 if exists("syntax_on")
@@ -78,3 +80,21 @@
 hi Underlined   guifg=#55ff55 guibg=#000000 gui=underline
 hi Error        guifg=#55ff55 guibg=#339933
 hi Todo         guifg=#113311 guibg=#44cc44 gui=none
+
+hi CursorColumn guibg=grey40
+hi CursorLine gui=NONE guibg=grey40
+hi Identifier gui=NONE
+hi Ignore guifg=bg
+hi MatchParen guibg=darkcyan
+hi MoreMsg gui=bold
+hi NonText gui=bold
+hi Pmenu guibg=magenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=darkgrey
+hi Question gui=bold
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi TabLine guibg=darkgrey guifg=NONE
