--- originals/moria.vim	2010-01-20 22:44:28.000000000 -0800
+++ patched/moria.vim	2010-04-21 18:43:41.000000000 -0700
@@ -1,3 +1,5 @@
+" VIM-XTERM-TAG
+
 if exists("g:moria_style")
     let s:moria_style = g:moria_style
 else
@@ -46,6 +48,7 @@
         hi CursorColumn ctermbg=DarkGray ctermfg=White guibg=#3a3a3a gui=none
         hi CursorLine ctermbg=DarkGray ctermfg=White guibg=#3a3a3a gui=none
     endif
+    set background=dark " Normal might change this
     if s:moria_monochrome == 1
         hi FoldColumn ctermbg=bg guibg=bg guifg=#a0a0a0 gui=none
         hi LineNr guifg=#a0a0a0 gui=none
@@ -103,11 +106,15 @@
         hi SpellCap guisp=#2c2cee gui=undercurl
         hi SpellLocal guisp=#2ceeee gui=undercurl
         hi SpellRare guisp=#ee2cee gui=undercurl
+        hi SpellBad ctermbg=88
+        hi SpellCap ctermbg=19
+        hi SpellRare ctermbg=90
+        hi SpellLocal ctermbg=30
     endif
     hi TabLineSel guibg=bg guifg=fg gui=bold
     hi Title ctermbg=Black ctermfg=White guifg=fg gui=bold
     if version >= 700
-        hi Visual ctermbg=LightGray ctermfg=Black guibg=#606060 gui=none
+        hi Visual ctermbg=LightGray ctermfg=Black guibg=#606060 guifg=NONE gui=none
     else
         hi Visual ctermbg=LightGray ctermfg=Black guibg=#606060 guifg=fg gui=none
     endif
@@ -142,6 +149,10 @@
     hi htmlItalic ctermbg=Black ctermfg=White guibg=bg guifg=fg gui=italic
     hi htmlUnderline ctermbg=Black ctermfg=White guibg=bg guifg=fg gui=underline
     hi htmlUnderlineItalic ctermbg=Black ctermfg=White guibg=bg guifg=fg gui=underline,italic
+
+    hi CursorColumn guifg=NONE
+    hi CursorLine guifg=NONE
+    hi Title guibg=NONE
 elseif &background == "light"
     if s:moria_style == "light"
         hi Normal ctermbg=White ctermfg=Black guibg=#f0f0f0 guifg=#000000 gui=none
@@ -154,6 +165,7 @@
         hi CursorColumn ctermbg=LightGray ctermfg=Black guibg=#dfdfdf gui=none
         hi CursorLine ctermbg=LightGray ctermfg=Black guibg=#dfdfdf gui=none
     endif
+    set background=light " Normal might change this
     if s:moria_monochrome == 1
         hi FoldColumn ctermbg=bg guibg=bg guifg=#7a7a7a gui=none
         hi Folded guibg=#cfcfcf guifg=#404040 gui=bold
@@ -203,11 +215,15 @@
         hi SpellCap guisp=#2c2cee gui=undercurl
         hi SpellLocal guisp=#008b8b gui=undercurl
         hi SpellRare guisp=#ee2cee gui=undercurl
+        hi SpellBad ctermbg=217
+        hi SpellCap ctermbg=153
+        hi SpellRare ctermbg=219
+        hi SpellLocal ctermbg=159
     endif
     hi TabLineSel guibg=bg guifg=fg gui=bold
     hi Title guifg=fg gui=bold
     if version >= 700
-        hi Visual ctermbg=LightGray ctermfg=Black guibg=#c4c4c4 gui=none
+        hi Visual ctermbg=LightGray ctermfg=Black guibg=#c4c4c4 guifg=NONE gui=none
     else
         hi Visual ctermbg=LightGray ctermfg=Black guibg=#c4c4c4 guifg=fg gui=none
     endif    
@@ -236,6 +252,9 @@
     hi htmlItalic ctermbg=White ctermfg=Black guibg=bg guifg=fg gui=italic
     hi htmlUnderline ctermbg=White ctermfg=Black guibg=bg guifg=fg gui=underline
     hi htmlUnderlineItalic ctermbg=White ctermfg=Black guibg=bg guifg=fg gui=underline,italic
+
+    hi CursorColumn guifg=NONE
+    hi CursorLine guifg=NONE
 endif
 
 hi! default link bbcodeBold htmlBold
