--- originals/moss.vim	2010-01-20 22:49:02.000000000 -0800
+++ patched/moss.vim	2010-04-21 18:45:08.000000000 -0700
@@ -6,6 +6,7 @@
 " Version: 2.0
 " URL: http://vim.sourceforge.net/script.php?script_id=2779
 " ------------------------------------------------------------------
+" VIM-XTERM-TAG
 
 " Init
 " ------------------------------------------------------------------
@@ -107,3 +108,13 @@
 hi Error       gui=NONE guifg=Khaki3 guibg=VioletRed4
 hi Todo        gui=BOLD guifg=GoldenRod3 guibg=NONE
 
+hi CursorLine gui=NONE
+hi DiffDelete gui=bold
+hi MoreMsg gui=bold
+hi NonText gui=bold
+hi Question gui=bold
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi lCursor guibg=fg guifg=bg
