--- originals/neverness.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/neverness.vim	2010-04-19 11:31:18.000000000 -0700
@@ -70,10 +70,10 @@
 hi Todo           ctermfg=LightMagenta     guifg=#9fb6cd guibg=#232323 gui=none
 
 " Spelling...
-hi SpellBad    ctermfg=DarkRed      ctermbg=black
-hi SpellCap    ctermfg=DarkBlue     ctermbg=black
-hi SpellRare   ctermfg=DarkYellow   ctermbg=black
-hi SpellLocal  ctermfg=DarkGreen    ctermbg=black
+hi SpellBad    ctermfg=160          ctermbg=16
+hi SpellCap    ctermfg=20           ctermbg=16
+hi SpellRare   ctermfg=220          ctermbg=16
+hi SpellLocal  ctermfg=34           ctermbg=16
 
 " "set cursorline"  and "set cursorcolumn" options.
 hi lCursor        guifg=#43705a guibg=#e6fff3 gui=none
@@ -139,3 +139,6 @@
 hi PmenuSbar   ctermfg=White ctermbg=LightCyan guifg=#ffffff guibg=#848484 gui=none
 hi PmenuThumb  ctermfg=White ctermbg=DarkGreen guifg=#ffffff guibg=#87ceeb gui=none
 
+hi MatchParen guibg=darkcyan
+hi SignColumn guibg=grey guifg=cyan
+hi TabLine guibg=darkgrey guifg=NONE
