--- originals/nightshimmer.vim	2010-01-20 22:53:46.000000000 -0800
+++ patched/nightshimmer.vim	2010-04-20 19:58:00.000000000 -0700
@@ -19,6 +19,7 @@
 
 """" GUI Colors
 
+highlight Normal        gui=None guibg=#103040 guifg=honeydew2
 highlight Cursor        gui=None guibg=Green guifg=White
 highlight CursorIM gui=bold guifg=white guibg=Green1
 highlight Directory     guifg=LightSeaGreen guibg=bg
@@ -43,7 +44,6 @@
     " Bottom fill (use e.g. same as LineNr)
     highlight NonText       gui=None guibg=grey6 guifg=Purple
 endif
-highlight Normal        gui=None guibg=#103040 guifg=honeydew2
 highlight Question      gui=bold  guifg=SeaGreen2 guibg=bg
 highlight Search        gui=NONE guibg=Purple4 guifg=NONE
 highlight SpecialKey    guibg=#123A4A guifg=#426272
@@ -81,9 +81,9 @@
 
 highlight PreProc       guifg=MediumPurple1
     "hi Include gui=None guifg=MediumPurple1 guibg=bg
-    "hi Define gui=None guifg=MediumPurple1g guibg=bg
-    "hi Macro gui=None guifg=MediumPurple1g guibg=bg
-    "hi PreCondit gui=None guifg=MediumPurple1g guibg=bg
+    "hi Define gui=None guifg=MediumPurple1 guibg=bg
+    "hi Macro gui=None guifg=MediumPurple1 guibg=bg
+    "hi PreCondit gui=None guifg=MediumPurple1 guibg=bg
 
 highlight Type          gui=NONE guifg=LightBlue
     "hi StorageClass gui=None guifg=LightBlue guibg=bg
@@ -109,3 +109,17 @@
 
 
 
+hi CursorColumn guibg=grey40
+hi CursorLine gui=NONE guibg=grey40
+hi Identifier gui=NONE
+hi MatchParen guibg=darkcyan
+hi Pmenu guibg=magenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=darkgrey
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi TabLine guibg=darkgrey guifg=NONE
+hi lCursor guibg=fg guifg=bg
