--- originals/PapayaWhip.vim	2010-04-12 21:24:23.000000000 -0700
+++ patched/PapayaWhip.vim	2010-04-21 21:34:25.000000000 -0700
@@ -7,6 +7,8 @@
 "
 " Only values that differ from defaults are specified.
 
+" VIM-XTERM-TAG
+
 set background=light
 hi clear
 if exists("syntax_on")
@@ -29,3 +31,44 @@
 hi VertSplit gui=None guibg=Gray
 hi Identifier guifg=#005f5f
 hi Statement ctermfg=DarkRed
+
+hi Cursor guibg=fg guifg=bg
+hi CursorColumn guibg=grey90
+hi CursorLine gui=NONE guibg=grey90
+hi DiffAdd guifg=NONE
+hi DiffChange guifg=NONE
+hi DiffDelete gui=bold
+hi DiffText guifg=NONE
+hi Directory guifg=blue
+hi Error guibg=red guifg=white
+hi ErrorMsg guibg=red guifg=white
+hi FoldColumn guibg=grey guifg=darkblue
+hi Folded guibg=lightgrey guifg=darkblue
+hi Ignore guifg=bg
+hi LineNr guifg=brown
+hi MatchParen guibg=cyan
+hi MoreMsg gui=bold guifg=seagreen
+hi NonText gui=bold
+hi Pmenu guibg=lightmagenta
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=grey
+hi Question gui=bold guifg=seagreen
+hi Search guibg=yellow
+hi SignColumn guibg=grey guifg=darkblue
+hi Special guifg=slateblue
+hi SpecialKey guifg=blue
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi Statement gui=bold guifg=brown
+hi StatusLine gui=bold,reverse
+hi TabLine guibg=lightgrey guifg=NONE
+hi Title gui=bold guifg=magenta
+hi Todo guibg=yellow guifg=blue
+hi Type gui=bold guifg=seagreen
+hi Underlined guifg=slateblue
+hi Visual guibg=lightgrey
+hi WarningMsg guifg=red
+hi WildMenu guibg=yellow guifg=black
+hi lCursor guibg=fg guifg=bg
