--- originals/pyte.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/pyte.vim	2010-04-13 11:36:30.000000000 -0700
@@ -90,3 +90,22 @@
 hi DiffDelete guifg=NONE guibg=#f0e0b0 gui=italic,bold
 
 
+hi CursorLine gui=NONE
+hi Directory guifg=blue
+hi ErrorMsg guibg=red guifg=white
+hi FoldColumn guibg=grey guifg=darkblue
+hi Ignore guifg=bg
+hi MoreMsg gui=bold guifg=seagreen
+hi NonText gui=bold
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=grey
+hi Question gui=bold guifg=seagreen
+hi Search guibg=yellow
+hi SignColumn guibg=grey guifg=darkblue
+hi SpellBad gui=undercurl guisp=red
+hi SpellCap gui=undercurl guisp=blue
+hi SpellLocal gui=undercurl guisp=darkcyan
+hi SpellRare gui=undercurl guisp=magenta
+hi Visual guibg=lightgrey
+hi WarningMsg guifg=red
+hi WildMenu guibg=yellow guifg=black
