--- originals/rdark.vim	2010-01-20 22:45:24.000000000 -0800
+++ patched/rdark.vim	2010-04-21 21:37:03.000000000 -0700
@@ -11,6 +11,8 @@
 "   0.5 - fixed the Pmenu colors
 "   0.6 - added SignColumn colors
 
+" VIM-XTERM-TAG
+
 set background=dark
 
 hi clear
@@ -23,9 +25,10 @@
 " Current Line
 if exists('rdark_current_line') && rdark_current_line == 1
 	set cursorline
-	hi CursorLine guibg=#191f21
 endif
 
+hi CursorLine guibg=#191f21
+
 " Default Colors
 hi Normal guifg=#babdb6 guibg=#1e2426
 hi NonText guifg=#2c3032 guibg=#2c3032 gui=none
@@ -157,3 +160,21 @@
 hi link cssTextAttr Constant
 hi link cssColorAttr Constant
 hi link cssFontAttr Constant
+
+hi Cursor guifg=bg
+hi CursorColumn guibg=grey40
+hi CursorLine gui=NONE
+hi DiffDelete gui=bold
+hi ErrorMsg guibg=red guifg=white
+hi Identifier gui=NONE
+hi Ignore guifg=bg
+hi MoreMsg gui=bold
+hi SignColumn guifg=cyan
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi Title gui=bold
+hi Underlined guifg=#80a0ff
+hi WarningMsg guifg=red
+hi lCursor guibg=fg guifg=bg
