--- originals/soso.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/soso.vim	2010-04-13 12:08:24.000000000 -0700
@@ -65,3 +65,29 @@
   hi TabLineSel                             gui=bold                                                       cterm=bold
   hi TabLineFill guifg=black  guibg=grey80                                                                 cterm=underline
 endif
+
+hi Cursor guibg=fg guifg=bg
+hi CursorColumn guibg=grey90
+hi CursorLine gui=NONE guibg=grey90
+hi DiffDelete gui=bold
+hi Directory guifg=blue
+hi Error guibg=red guifg=white
+hi ErrorMsg guibg=red guifg=white
+hi FoldColumn guibg=grey guifg=darkblue
+hi Folded guibg=lightgrey guifg=darkblue
+hi Identifier guifg=darkcyan
+hi Ignore guifg=bg
+hi MoreMsg gui=bold guifg=seagreen
+hi PmenuSel gui=NONE
+hi Question gui=bold guifg=seagreen
+hi SignColumn guibg=grey guifg=darkblue
+hi SpellLocal gui=undercurl guisp=darkcyan
+hi SpellRare gui=undercurl guisp=magenta
+hi TabLine guifg=NONE
+hi TabLineFill gui=reverse
+hi Title gui=bold guifg=magenta
+hi Todo guibg=yellow guifg=blue
+hi Underlined guifg=slateblue
+hi WarningMsg guifg=red
+hi WildMenu guibg=yellow guifg=black
+hi lCursor guibg=fg guifg=bg
