--- originals/tabula.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/tabula.vim	2010-04-20 11:53:18.000000000 -0700
@@ -24,8 +24,8 @@
 "   such that text files be displayed other than e.g. c sources
 " =============================================================================
 
-hi clear
 set background=dark
+hi clear
 if exists("syntax_on")
     syntax reset
 endif
@@ -165,6 +165,8 @@
 "                             Color Definitions                            {{{1
 "==============================================================================
 
+hi Normal               guifg=#71D289   guibg=#004A41                   ctermfg=84      ctermbg=23 
+
 if version >= 700
     hi SpellBad         guisp=#FF0000
     hi SpellCap         guisp=#afaf00
@@ -275,8 +277,6 @@
 hi ModeMsg              guifg=#FFFFFF   guibg=#0000FF   gui=NONE        ctermfg=7       ctermbg=4       cterm=NONE
 hi MoreMsg              guifg=#FFFFFF   guibg=#00A261   gui=NONE        ctermfg=7       ctermbg=28      cterm=NONE
 
-hi Normal               guifg=#71D289   guibg=#004A41                   ctermfg=84      ctermbg=23 
-
 "------------------------------------------------------------------------------
 " Preprocessor Variants:
 "------------------------------------------------------------------------------
@@ -455,6 +455,17 @@
 hi BadWord              guifg=#E4E300                   gui=underline   ctermfg=11                      cterm=underline
 
 
+hi CursorLine gui=NONE
+hi DiffDelete gui=bold
+hi NonText gui=bold
+hi SpellBad gui=undercurl guisp=#ff0000
+hi SpellCap gui=undercurl guisp=#afaf00
+hi SpellLocal gui=undercurl guisp=#00afaf
+hi SpellRare gui=undercurl guisp=#bf4040
+hi Underlined guifg=#80a0ff
+hi lCursor guibg=fg guifg=bg
+hi pythonPreCondit guifg=NONE
+
 "==============================================================================
 "                              Options Processor                           {{{1
 "==============================================================================
