--- originals/tcsoft.vim	2010-01-20 23:43:36.000000000 -0800
+++ patched/tcsoft.vim	2010-04-20 20:34:18.000000000 -0700
@@ -10,7 +10,7 @@
 if exists("syntax_on")
   syntax reset
 endif
-let colors_name = "TCSoft"
+let colors_name = "tcsoft"
 
 if version >= 700
   hi CursorLine                 guibg=#FFFF33  gui=NONE  "hellgelb
@@ -81,3 +81,28 @@
 hi link   Delimiter	  Special
 hi link   SpecialComment Comment
 hi link   Debug		    Special
+
+hi Cursor guibg=fg
+hi DiffAdd guibg=lightblue
+hi DiffChange guibg=lightmagenta
+hi DiffDelete gui=bold guibg=lightcyan guifg=blue
+hi DiffText guibg=red
+hi Directory guifg=blue
+hi Error guibg=red guifg=white
+hi FoldColumn guibg=grey guifg=darkblue
+hi Folded guibg=lightgrey guifg=darkblue
+hi NonText gui=bold guifg=blue
+hi PmenuSbar guibg=grey
+hi PmenuSel guibg=grey
+hi Search guibg=yellow
+hi SignColumn guibg=grey guifg=darkblue
+hi SpecialKey guifg=blue
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi Title gui=bold guifg=magenta
+hi Underlined guifg=slateblue
+hi Visual guibg=lightgrey
+hi WildMenu guibg=yellow guifg=black
+hi lCursor guibg=fg guifg=bg
