--- originals/tir_black.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/tir_black.vim	2010-04-12 11:43:09.000000000 -0700
@@ -128,3 +128,26 @@
 
 " Special for CSharp
 hi link csXmlTag Keyword 
+
+hi DiffAdd guibg=darkblue
+hi DiffChange guibg=darkmagenta
+hi DiffDelete gui=bold guibg=darkcyan guifg=blue
+hi DiffText guibg=red
+hi Directory guifg=cyan
+hi Error guibg=red guifg=white
+hi FoldColumn guibg=grey guifg=cyan
+hi Identifier gui=NONE
+hi Ignore guifg=bg
+hi MoreMsg gui=bold guifg=seagreen
+hi NonText gui=bold
+hi Question gui=bold guifg=green
+hi SignColumn guibg=grey guifg=cyan
+hi SpellBad gui=undercurl guisp=red
+hi SpellCap gui=undercurl guisp=blue
+hi SpellLocal gui=undercurl guisp=cyan
+hi SpellRare gui=undercurl guisp=magenta
+hi Statement gui=bold
+hi TabLine guibg=darkgrey guifg=NONE
+hi Type gui=bold
+hi Underlined guifg=#80a0ff
+hi lCursor guibg=fg guifg=bg
