--- originals/vylight.vim	2010-04-05 10:31:09.000000000 -0700
+++ patched/vylight.vim	2010-04-13 12:50:45.000000000 -0700
@@ -79,3 +79,28 @@
 hi link SpecialComment  Special
 hi link Debug           Special
 
+hi Cursor guifg=bg
+hi CursorColumn guibg=grey90
+hi DiffAdd guibg=lightblue
+hi DiffChange guibg=lightmagenta
+hi DiffDelete gui=bold guibg=lightcyan guifg=blue
+hi DiffText guibg=red
+hi Directory guifg=blue
+hi FoldColumn guibg=grey guifg=darkblue
+hi Folded guibg=lightgrey guifg=darkblue
+hi Ignore guifg=bg
+hi MoreMsg gui=bold guifg=seagreen
+hi NonText gui=bold
+hi Question gui=bold guifg=seagreen
+hi SignColumn guibg=grey guifg=darkblue
+hi SpecialKey guifg=blue
+hi SpellBad gui=undercurl guisp=red
+hi SpellCap gui=undercurl guisp=blue
+hi SpellLocal gui=undercurl guisp=darkcyan
+hi SpellRare gui=undercurl guisp=magenta
+hi TabLine guibg=lightgrey guifg=NONE
+hi Title gui=bold
+hi Underlined guifg=slateblue
+hi WarningMsg guifg=red
+hi WildMenu guibg=yellow guifg=black
+hi lCursor guibg=fg guifg=bg
