--- originals/xoria256.vim	2009-06-04 20:53:30.000000000 -0700
+++ patched/xoria256.vim	2010-04-20 20:41:09.000000000 -0700
@@ -91,4 +91,18 @@
 hi diffChange   ctermfg=bg  guifg=bg      ctermbg=181 guibg=#dfafaf
 hi diffText     ctermfg=bg  guifg=bg      ctermbg=174 guibg=#df8787 cterm=none gui=none
 
+hi Cursor guifg=bg
+hi CursorLine gui=NONE
+hi Directory guifg=cyan
+hi MoreMsg gui=bold guifg=seagreen
+hi Question gui=bold guifg=green
+hi SignColumn guibg=grey
+hi SpellBad guibg=NONE
+hi SpellCap guibg=NONE
+hi SpellLocal guibg=NONE
+hi SpellRare guibg=NONE
+hi Title gui=bold guifg=magenta
+hi WarningMsg guifg=red
+hi lCursor guibg=fg guifg=bg
+
 " vim: set expandtab tabstop=2 shiftwidth=2 smarttab softtabstop=2:
